### DISCUSSION TITLE
1.10 Update - Crop, Folders, Transform Multiple Drawings, Canvas Flip, Out of Pegs, New Pencils…

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#418] [Announcements]
ToonSquid 1.10 is out now and it comes with many exciting improvements. Here are the most important ones:

- Crop: Change the canvas size after a project or animation clip has been created via Settings → Info → Crop.
- Folders: Organize your projects and assets using folders in the library.
- Edit multiple selected drawings at once using the transform tool.
- New pencil brushes: "Sketch Pencil" and "Sandy"
- New brush settings: Tilt Size Compression, Tilt Control Stretch, Tilt Control Shift, Gradation Tilt Control, Bleed Tilt Control
- The tilt threshold is now a property of each brush and not a global setting anymore.
- Updated the following brushes: "Pencil", "6B Pencil", "Flat Brush", "Blotchy", "Dry Watercolor". You need to manually reset those brushes if you want the updates to be applied to them.
- Polygon Selection: The freehand selection tool mode now allows you to tap individual points to create polygonal selections and doesn't close the selection region automatically anymore. The old behaviour can be restored via Settings → Tools → Selection → Manually close selections.
- Canvas Flip: Temporarily flip the canvas horizontally and vertically (Settings → General).
- Out of Pegs: Also known as "Shift and Trace", this feature lets you temporarily move your onion skins to help you draw inbetweens while keeping consistent proportions.
- Use the "Select multiple" action as an alternative to the double-tap-and-drag gesture to select multiple drawings or keyframes in the timeline. The gesture can now also be disabled in the gesture settings.
- Reverse Drawings: Multiple selected drawings in the same layer can now be reversed using the "Reverse Drawings" action.
- ProRes 4444: When exporting an MOV video with transparency included, you can now optionally choose the ProRes 4444 codec, which has better compatibility with some third-party software than HEVC.
- Camera Outside Darkening: The outside regions of the camera layer are now darkened during playback to make it easier to see what will be visible in the final export and what won't. This can also be enabled during editing and the darkening amount is configurable via the inspector.
- Temporary Pivot: Enable and edit a temporary pivot point when editing layer contents instead of the transform properties.
- You can now disable the quick pinch gesture for resetting the canvas zoom.
- The pinch-to-zoom gesture in the timeline now feels a lot more reliable.
- The font "Weight" and "Style" dropdowns have been combined into a single dropdown that now contains all font variants of an installed typeface.
- Left Handed UI mode
- Various bug fixes
- The minimum supported iPadOS version is now 15.0. (All iPad models running iPadOS 13 and 14 can be updated to iPadOS 15)

**Comment by DWrightAnimation** [comment]
I'm ecstatic that Out of Pegs has been added. Do you think we could get "Fixed Onion Skin Toggle" next? It's a feature in OpenToonz that lets you set a drawing or frame as a pinned onion skin, so no matter how far you are from it, it'll stay in the onion skin. Kind of like continuously using one of the drawings for an onion skin on animation paper.

**Comment by trinhpxl** [comment]
GUYS AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA IM SO HYPE FOR THIS NEW UPDATE TYSM KEIWAN FOR UR HARD WORK KEEP AT IT RAAAAAAAAAAAAAAAAA

**Comment by yaknepper** [comment]
What an absolute awesome update. Thank you very much!

**Comment by DWrightAnimation** [comment]
Was the "Temporary Pivot Point" option for selections always there, or did that just get added?

**Reply by keiwando** [reply]
It's new in 1.10. I show it in the YouTube video, but must have forgotten to add it to the release notes

**Comment by mshoek** [comment]
THANK YOU KEIWANDO!!! Out-of-pegs is a lifesaver! And I really appreciate the other features as well!

![thankyou-dwight](https://github.com/keiwando/toonsquid/assets/118011378/8e433f3b-05a6-4643-b141-63551952ca44)

**Comment by andydjames** [comment]
Awesome updates. The timeline is a lot more responsive when zooming and select multiple makes selections a bit easier. Folders are a great addition but drag and dropping a folder deletes it (thankfully not the contents). Out of pegs looks great too.
The canvas flip feels like it should belong on the main UI.
You're smashing it, Keiwando. Thank you for all your hard work.
